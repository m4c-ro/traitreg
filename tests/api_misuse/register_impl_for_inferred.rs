fn main() {}



trait MyTrait {}

#[traitreg::register]
impl MyTrait for _ {}
