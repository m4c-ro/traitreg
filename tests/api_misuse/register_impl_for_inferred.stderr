error: custom attribute panicked
 --> tests/api_misuse/register_impl_for_inferred.rs:7:1
  |
7 | #[traitreg::register]
  | ^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: Cannot register implementation on an inferred type (_), expected a struct, enum, union or type alias.
